packages/server/graphql/meetingTypePredicates.ts

```diff
@@ -0,0 +1,20 @@
+import MeetingPoker from '../database/types/MeetingPoker'
+import MeetingRetrospective from '../database/types/MeetingRetrospective'
+import MeetingAction from '../database/types/MeetingAction'
+import GenericMeetingStage from '../database/types/GenericMeetingStage'
+import EstimateStage from '../database/types/EstimateStage'
+
+export const isRetroMeeting = (
+  meeting: MeetingPoker | MeetingRetrospective | MeetingAction
```

Comment:
+2 love the predicates!

+1 can you create a union type like `type AnyMeeting = MeetingPoker | MeetingRetrospective | MeetingAction`? this'll probably be stored in a pg query when we refactor the meeting table, but for now it could live in rethinkDriver.ts or wherever you think is best

Reply:
I'm happy to do so, just one more question.
In `rethinkDriver` we already have the following
```
  NewMeeting: {
    type: MeetingRetrospective | MeetingAction | MeetingPoker
    index: 'facilitatorUserId' | 'teamId' | 'templateId'
  }
```
So currently when we fetch the meeting from db we get the union type. By adding `AnyMeeting` do we want to future proof a bit or what? I'm happy to add it, just making sure what's the purpose as for today 

BTW, do avoid more conflicts I'm going to merge this and I'll submit a little PR after clarifying the above, hope you don't mind 🙏🏻